### PR TITLE
Added lingering injuries tables

### PR DESCRIPTION
**Pull request by joshwilliams6**

### CODE REVIEW

**Comment by joshwilliams6** [comment]
force-push was to fix injury descriptions referencing other descriptions.

File: src/Tables/Injury/thunder.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+using System.Linq;
+using Generator.Models;
+using Toolkit.Generator;
+
+namespace Generator.Tables.Injury
+{
+    public class ThunderInjury : ITable
+    {
+        private Percentile[] _table = {
+            new Percentile(0,4,"*Brain Injury.* You have suffered a brain injury. You have disadvantage on Intelligence, Wisdom, and Charisma checks, as well as Intelligence, Wisdom, and Charisma saving throws. If you fail a saving throw against bludgeoning damage, force damage, or psychic damage, you are also stunned until the end of your next turn. Magic such as the regenerate spell can restore your full brain function.),
```
Comment (Randy-Mills) [inline src/Tables/Injury/thunder.cs:11]: Missing closing "

File: src/Commands/MadnessCommand.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                if (re.IsMatch(element))
+                {
+                    randomValue = Int32.Parse(element);
+                    break;
+                }
+            }
+
+            if (randomValue > table.Max)
+                return $"Provided value is out of range. Selected table has {table.Max} rows.";
+        }
```
Comment (Randy-Mills) [inline src/Commands/MadnessCommand.cs:54]: Needs a final return for happy path.

return $">>> [{randomValue}]: {table.Fetch(randomValue)}"

or something similar.

File: src/Commands/MadnessCommand.cs [hunk 3]
```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Text.RegularExpressions;
+using Generator.Tables.Injury;
```
Comment (Randy-Mills) [inline src/Commands/MadnessCommand.cs:3]: Should be `using Generator.Tables.Madness;` instead of Injury here.

**Comment by Randy-Mills** [comment]
Outside of the small issues raised PR looks good. Tested with fixes locally and everything worked as expected. @joshwilliams6 once those changes are pushed up I'll merge it!

**Review by Randy-Mills — APPROVED** [review]

**Pull request merged**